packages/stores/src/ui-config/manage-liquidity/add-concentrated-liquidity.ts

```diff
@@ -446,6 +446,27 @@ export class ObservableAddConcentratedLiquidityConfig extends TxChainSetter {
     );
   }
 
+  @computed
+  get volatilityType(): "passive" | "custom" | "aggressive" | "moderate" {
+    const isRangePassive = this.fullRange;
+    const isRangeAggressive =
+      !isRangePassive &&
+      this.tickRange[0].equals(this.aggressiveTickRange[0]) &&
+      this.tickRange[1].equals(this.aggressiveTickRange[1]);
+    const isRangeModerate =
+      !isRangePassive &&
+      this.tickRange[0].equals(this.moderateTickRange[0]) &&
+      this.tickRange[1].equals(this.moderateTickRange[1]);
+    const isRangeCustom =
+      !isRangeAggressive && !isRangeModerate && !isRangePassive;
+
+    if (isRangePassive) return "passive";
+    if (isRangeModerate) return "moderate";
+    if (isRangeAggressive) return "aggressive";
+    if (isRangeCustom) return "custom";
+    return "custom";
+  }
```

Comment:
This is a perfect example of something that should be in a UI config. 

Thoughts on changing name to `strategyPresets` ? Low prio